README.md

```diff
@@ -1,3 +1,12 @@
+# Consider using confluentinc/confluent-kafka-dotnet
+
+kafka-sharp was built at a time where there was no other .NET Kafka driver
+available. As the Kafka protocol evolved a lot, it became costly to maintain.
+Also, new use-cases arised at Criteo for which this software was not design
+for. For these reasons, we've started to migrate to
```

Comment:
For these reasons, Criteo migrated to [...] and this project is no longer maintained.